Index: toys/mug/Makefile.in
===================================================================
--- toys/mug/Makefile.in.orig
+++ toys/mug/Makefile.in
@@ -336,7 +336,7 @@ TEST_PROGS =
 # enforce compiling this dir first before decending into tests/
 SUBDIRS = .
 AM_CPPFLAGS = -I${top_srcdir} -I${top_srcdir}/lib $(GTK_CFLAGS) $(WEBKIT_CFLAGS) \
-	-DICONDIR='"$(icondir)"' -DMUGDIR='"$(abs_srcdir)"' \
+	-DICONDIR='"$(icondir)"' -DMUGDIR='"$(DATADIR)"' \
 	-DGTK_DISABLE_DEPRECATED -DGSEAL_ENABLE
 
 
Index: toys/mug/Makefile.am
===================================================================
--- toys/mug/Makefile.am.orig
+++ toys/mug/Makefile.am
@@ -20,7 +20,7 @@ include $(top_srcdir)/gtest.mk
 SUBDIRS= .
 
 AM_CPPFLAGS=-I${top_srcdir} -I${top_srcdir}/lib $(GTK_CFLAGS) $(WEBKIT_CFLAGS) \
-	-DICONDIR='"$(icondir)"' -DMUGDIR='"$(abs_srcdir)"' \
+        -DICONDIR='"$(icondir)"' -DMUGDIR='"$(DATADIR)"'  \
 	-DGTK_DISABLE_DEPRECATED -DGSEAL_ENABLE
 
 # don't use -Werror, as it might break on other compilers
